SwiftWisdom/Core/Geometry/Geometry.swift

```diff
@@ -86,6 +86,11 @@ public func random(range: Range<Int>) -> Int {
 }
 
 public func avg(numbers: CGFloat...) -> CGFloat {
+    return avg(numbers)
+}
+
+public func avg(numbers: [CGFloat]) -> CGFloat {
```

Comment:
I think there's some value to having `avg` as a global function as well, but it might be better to also move it to a generic implementation for greater flexibility and usefulness.